tests/cli/fixtures/multi_provider_config.yaml

```
@@ -0,0 +1,24 @@
+$schema: ../../../../mcp-agent/schema/mcp-agent.config.schema.json
+
+execution_engine: asyncio
+logger:
+  transports: [console, file]
+  level: debug
+
+# Multiple model providers with API keys
+openai:
+  default_model: gpt-4o
+  api_key: !developer_secret ${oc.env:OPENAI_API_KEY}
```

I think some of these fixtures aren't actually valid right now. We moved off this `${oc.env:ENV_VAR}` format to just directly inlining ENV_VAR after the tag
```suggestion
  api_key: !developer_secret OPENAI_API_KEY
```